site/scripts/seed-db.ts

```diff
@@ -66,6 +66,11 @@ const script = async () => {
       preferredName: "Alice",
       email: "alice@example.com",
     },
+    {
+      shortname: "hash",
+      preferredName: "HASH",
+      email: "hash@example.com",
+    },
```

Comment:
I was getting 404 when going to `localhost:3000/@hash`, even after seeding the data. Adding this change makes it easy to get started locally for everyone now. Happy to remove the diff I’ve missed some instructions.

Reply:
You're right. I think this is a good addition to avoid local development confusion.